Lesson1/Denis Sulima/task1/main.cpp

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+#include <vector>
+std::vector<int> compoundVectors(std::vector<int>::iterator vectorFirstBegin, std::vector<int>::iterator vectorFirstEnd,
+                                std::vector<int>::iterator vectorSecondBegin, std::vector<int>::iterator vectorSecondEnd) {
+    std::vector<int>tmp {vectorFirstBegin,vectorFirstEnd};
+    tmp.insert(tmp.end(), vectorSecondBegin, vectorSecondEnd);
+    return tmp;
+}
```

Comment:
Nice approach.
Try to estimate the algorithm complexity by time and memory in big O notation.